rssfeed/models.py

```diff
@@ -17,37 +18,27 @@ def __unicode__(self):
         return self.title or self.xml_url
 
     def save(self, *args, **kwargs):
-        """Poll new Feed"""
+        """Poll new Feed """
```

Comment:
Something this short, with not a lot of info to it might be better suited to a normal comment.